scss/_patterns_accordion.scss

```diff
@@ -1,5 +1,11 @@
 @import 'settings';
 
+@mixin align-icon($icon-size: 1rem, $line-height: default-text) {
```

Comment:
Just a little suggestion to maybe rename it to `accordion-align-icon` to be a bit safer about not causing any name collisions.

Reply:
With the new update I'm no longer using a mixin, but good point :+1: